### ISSUE TITLE
Create git branch ?

### ISSUE THREAD

**Issue by YiuRULE** [nvareille/GameProjectII#1]
On avait pas trop parlé de ça, donc j'ai push sur la master pour que pour vous puissiez commencer votre taff', mais ça serait cool qu'on crée des branches pour chaque parties du projet ?

**Comment by nvareille** [comment]
Oui, une pour chaque fonctionnalité majeure tant qu'elle n'a pas été testée
